contents/docs/feature-flags/local-evaluation.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Server-side local evaluation
+sidebar: Docs
+showTitle: true
+availability:
+    free: full
+    selfServe: full
+    enterprise: full
+---
+
+> **Note:** Local evaluation is only available in the [Node](/docs/libraries/node), [Ruby](https://posthog.com/docs/libraries/ruby), [Go](https://posthog.com/docs/libraries/go), and [Python](https://posthog.com/docs/libraries/python) SDKs.
+
+> **Note:** Do not use local evaluation in an edge / lambda environment, as this can slow down performance, and also raise your bill drastically. It's best to use regular flag evaluation instead.
```

Comment:
Having two separate boxouts at the top is jarring. Can we find a better solution here?